src/engine/resource_manager/mod.rs

```diff
@@ -370,7 +370,8 @@ impl ResourceManagerState {
         containers.curves.destroy_unused();
     }
 
-    pub(in crate) fn update(&mut self, dt: f32) {
+    /// Reload resources if they have changed in disk.
```

Comment:
I think this comment is misleading, what `update` really does is it updates resource containers *and* performs hot reloading. Update of resource container removes resources that weren't used for some time (afaik 60 seconds by default).

Also it is worth noting that this method should not be called manually if one use `Engine::update`